kubejs/server_scripts/ae2/tags.facades.js

```diff
@@ -0,0 +1,20 @@
+// priority: 0
+"use strict";
+
+function registerAe2FacadeWhitelistTags(event) {
+    
+    const createdeco_catwalks_fm =
```

Comment:
might be worth adding this to startup_scripts/create_deco/constants.js instead of duplicating it twice?